taglib-sharp.pc.in

```diff
@@ -0,0 +1,8 @@
+prefix=@prefix@
+exec_prefix=${prefix}
+libdir=${exec_prefix}/lib
+
+Name: TagLib Sharp
+Description: A library for reading and writing tags to audio files.
```

Comment:
Let's change this to:
 A .NET platform-independent library for reading and writing metadata in media files, including video, audio, and photo formats.